src/client/features/interactions/interactions-menu.js

```diff
@@ -40,6 +53,8 @@ class InteractionsMenu extends React.Component {
       edges.removeClass('type-hidden');
     }
 
+    this.hideTooltips();
```

Comment:
You should just emit an event here.
i.e.

```js
cy.emit('hide-type');
```
Then, in bindEvents, do something like:
```js
cy.on('hide-type', () => {
  hideTooltips();
});
```